frontend/app/views/userprofile.html

```diff
@@ -1,3 +1,5 @@
 <div ng-controller="UserProfileCtrl as userprofile">
   User Profile panel.
+  <button type="button" ng-click="leaveTrainingFeedback()" ng-dialog-class="ngdialog-theme-plain">SMTH1</button>
```

Comment:
I understand that this is temporary button, but use more understandable names. Thanks!